pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/RuntimeUtils.java

```diff
@@ -355,6 +357,14 @@
 
         args.add("--pulsar_serviceurl");
         args.add(pulsarServiceUrl);
+        if (instanceConfig.getFunctionDetails().getRuntime() == Function.FunctionDetails.Runtime.JAVA) {
+            // TODO: for now only Java function context exposed pulsar admin, so python/go no need to pass this argument
+            // until pulsar admin client enabled in python/go function context.
+            if (pulsarWebServiceUrl != null) {
```

Comment:
we need to add a flag in the function worker config to control whether to expose PulsarAdmin through context. Because the new function worker will add this `--web_serviceurl` to the command line that is not able to recognize by the function instance. So we need to have a flag to allow people to do the following update sequence:

1. Update function worker with exposing pulsar admin disabled.
2. Update all the functions to the new image that understand `--web_serviceurl`
3. Update function worker to enable exposing pulsar admin.

Reply:
I agree, flag added, also add some tests when `exposePulsarAdminClientEnabled=false`